src/routes/MemberProfile/MemberProfile.js

```diff
@@ -218,6 +223,32 @@ export function ActionDropdown ({ items }) {
     />
 }
 
+export function ProjectsSection ({ memberCap, projects, showDetails }) {
+  return (
+    <div>
+      <div styleName='profile-subhead'>
```

Comment:
I think this heading would be better in the parent markup as done with the SkillsSection? This heading so far seems to me seems business of the MemberProfile layout. This may eliminate the use for this `ProjectsSection` component entirely which I think isn't a bad thing...